CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 - On macOS, updated core-* dependencies and cocoa.
 - Bump `parking_lot` to 0.11
 - On Android, bump `ndk`, `ndk-sys` and `ndk-glue` to 0.2. Checkout the new ndk-glue main proc attribute.
+- **Breaking:** `Fullscreen` enum now uses `Option<Borderless>` instead of `Borderless` to pick system preferred monitor.
```

Comment:
`Borderless(Option<MonitorHandle>)`, not `Option<Borderless>`, right?

Reply:
indeed.